### ISSUE TITLE
Visual Studio 2019 16.7.2 keeps disabling XamlStyler

### ISSUE THREAD

**Issue by HakanCelikMK1** [Xavalon/XamlStyler#297]
First of all, I would like to thank to the people who started, kept alive and contributed to this incredible extension. It has become insdipensable for my Xamarin.Forms development.

I think my issue is most likely related to Visual Studio itself, instead of Xaml Styler, but since there are very knowledgeable people here, I thought perhaps I could get proper help here instead of that dreadful VS Developer Community.

The problem started (I think) when I  upgraded VS 2019 from 16.7.0 to 16.7.1. All my 3rd party extensions became disabled. I've tried to enable them to no avail. 

Then, I uninstalled and reinstalled them and everything returned to normal, except Xaml Styler. Some of my working extensions are : Copy As Html, Extension Manager 2019, Format All Files, Indent Guides, Text Sharp etc.

Whatever I do, I can't enable it.  I tried everything I could find on the Net (cleaning the extension cache, resetting the settings etc.), but no joy there. I really don't have time to uninstall VS and reinstall it with all the extensions, components etc.

I'm using VS in admin mode, but installing in admin or normal mode didn't make any difference.

Here is hoping you could give me some pointers, checking the logs I've attached.
Regards, Hakan.

**Describe the bug**
Visual Studio 2019 16.7.2 keeps disabling XamlStyler. VS Log says the extension is successfully loaded but it doesn't say it is enabled, like it does for other extensions.

Successfully loaded extension 'XAML Styler' (0c572937-0d4a-48cc-aa3f-875ffa9805ba)...

**Expected behavior**
After installation I expect the extension to be enabled and usable

**Version Info :**
 - OS: Microsoft Windows 10 Enterprise Version 1909, Build : 18363.1016
 - Visual Studio: Microsoft Visual Studio Professional 2019 (16.7.2+30413.136)
 - XAML Styler: Extension (3.2003.9)

**Logs :**
[VSIXInstaller.log](https://github.com/Xavalon/XamlStyler/files/5113824/VSIXInstaller.log)
[VSLog.xml.log](https://github.com/Xavalon/XamlStyler/files/5113825/VSLog.xml.log)
[ActivityLog.Setup.xml.log](https://github.com/Xavalon/XamlStyler/files/5113827/ActivityLog.Setup.xml.log)
[VSLogAfterEnable.xml.log](https://github.com/Xavalon/XamlStyler/files/5113849/VSLogAfterEnable.xml.log)

**Comment by HakanCelikMK1** [comment]
Hi,
When I noticed that a new version is released, I decided to try that right away.
I uninstalled the old extension and installed the latest one. Yay, IT'S ALIVE, everything is working as expected for the time being.
For reference, I'm attaching the uninstall log and the VS log from the first run after the installation of the latest version.

I leave it up to you to decide whether this issue should be closed or not.

Regards, Hakan.

[UninstallLog.xml.log](https://github.com/Xavalon/XamlStyler/files/5146852/UninstallLog.xml.log)
[ActivityLog.Setup.xml.log](https://github.com/Xavalon/XamlStyler/files/5146853/ActivityLog.Setup.xml.log)
[VSFirstRunLog.xml.log](https://github.com/Xavalon/XamlStyler/files/5146854/VSFirstRunLog.xml.log)
